tests/phpunit/CRM/Activity/BAO/ActivityTest.php

```diff
@@ -757,23 +757,18 @@ public function testbyActivityDateAndStatus() {
     $lastWeekActivities = array(1, 2, 3);
     $todayActivities = array(4, 5, 6, 7);
     $lastTwoMonthsActivities = array(8, 9, 10, 11);
-    $lastYearActivties = array(12, 13, 14, 15, 16);
```

Comment:
I think this should be in a separate PR - even if we have to ignore the test fail for now